### PR TITLE
Support moving project cards between columns

### PR DESCRIPTION
**Pull request by aidy**
A bit hacky. Github API doesn't allow for easy retrieval of the
project card id given an issue id.

### CODE REVIEW

**Comment by dan-slinky-ckpd** [comment]
@takanabe please can you review 🙏

**Comment by takanabe** [comment]
👀

File: main.go [hunk 1]
```diff
@@ -269,3 +287,28 @@ func validateGitHubResponse(res *github.Response, err error) error {
...
+	for _, event := range events {
+		if event.ProjectCard != nil {
+			cards = append(cards, event.ProjectCard)
+		}
+	}
+	return cards, nil
+}
+
+func moveCardInProject(ctx context.Context, client *github.Client, columnID int64, card *github.ProjectCard) error {
+	resp, err := client.Projects.MoveProjectCard(ctx, *card.ID, &github.ProjectCardMoveOptions{ColumnID: columnID, Position: "bottom"})
```
Comment (takanabe) [inline main.go]: Let's put the new cards at the top of the specified column.

```suggestion
	resp, err := client.Projects.MoveProjectCard(ctx, *card.ID, &github.ProjectCardMoveOptions{ColumnID: columnID, Position: "top"})
```

**Comment by takanabe** [comment]
@aidy Thank you for your PR! 

>A bit hacky. Github API doesn't allow for easy retrieval of the
project card id given an issue id.

Yeah, we cannot easily retrieve relations between project cards and issues unfortunately with REST API.

Could you elaborate on the expected situation when you want to move cards between columns except for the creation time? For example, do you want to move cards when the labels on the existing issues are changed?

**Comment by aidy** [comment]
> Could you elaborate on the expected situation when you want to move cards between columns except for the creation time? 

It implements the readme - https://github.com/takanabe/github-actions-automate-projects/blame/master/README.md#L94

If you have actions for open and also for edit/close, then it currently fails as it tries to create a new project card.

(Specifically we have a use case for moving certain kinds of closed issues to a sprint review column)

**Review by takanabe — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (aidy) [reply main.go]: wfm - I'll rebase into the original commit

**Pull request merged**